components/invitation_modal/invitation_modal_members_step.jsx

```diff
@@ -68,6 +88,18 @@ export default class InvitationModalMembersStep extends React.Component {
 
     render() {
         const inviteUrl = getSiteURL() + '/signup_user_complete/?id=' + this.props.inviteId;
+        let copyLinkConfirm = null;
+        if (this.state.copiedLink) {
+            copyLinkConfirm = (
+                <p className='alert alert-success alert--confirm'>
+                    <SuccessIcon/>
+                    <FormattedMessage
+                        id='get_link.clipboard'
```

Comment:
no changes to i18n ?

Reply:
This comes from other part of the code, concretely from `components/get_link_modal.jsx`